changelog/59276.fixed

```diff
@@ -0,0 +1 @@
+When checking if the mode had changed, only do so if the passed mode is not None.
```

Comment:
Can you elaborate for which module, to make it easier for users reading the changelog